pyface/split_panel.py

```diff
@@ -22,11 +23,24 @@ class SplitPanel(Widget, SplitWidget):
     # 'object' interface.
     # ------------------------------------------------------------------------
 
-    def __init__(self, parent, **traits):
+    def __init__(self, parent=None, **traits):
         """ Creates a new panel. """
 
-        # Base class constructor.
-        super().__init__(**traits)
+        create = traits.pop("create", True)
 
-        # Create the widget's toolkit-specific control.
-        self.control = self._create_splitter(parent)
+        # Base class constructor.
+        super().__init__(parent=parent, **traits)
+
+        if create:
+            # Create the widget's toolkit-specific control.
+            self.create()
+            warnings.warn(
+                "automatic widget creation is deprecated and will be removed "
+                "in a future Pyface version, use create=False and explicitly "
+                "call create() for future behaviour",
+                PendingDeprecationWarning,
+            )
```

Comment:
same comment here regarding an issue to handle this deprecation - looks like a single issue will suffice to handle all such deprecations related to the `create` method.

Reply:
Same reply as before - I've updated #729 to add the sequence around this.

Actually, should probably add #729 to the 8.0 milestone.